src/controllers/userController.js

```diff
@@ -67,6 +101,19 @@ const updateCurrentUser = async (req, res) => {
 };
 
 const updateUserPassword = async (req, res) => {
+  /*
+     #swagger.summary = 'Fetch a user by id, update their password and save it to the database.'
+     #swagger.parameters['id'] = {
+        description: 'Mongo ObjectID of the user to fetch',
+     }
```

Comment:
This endpoint does not take any query or path parameters; it just uses the cookie to find the requesting user's id.